mhs/mhs/common/messages/ebxml_request_envelope.py

```diff
@@ -60,10 +77,14 @@ def _parse_mime_message(headers: Dict[str, str], message: str) -> email.message.
         :return: a Message that represents the message received.
         """
         content_type = headers[CONTENT_TYPE_HEADER_NAME]
-        content_type_header = CONTENT_TYPE_HEADER_NAME + ": " + content_type + "\r\n"
+        content_type_header = CONTENT_TYPE_HEADER_NAME + ": " + content_type + "\r\n\r\n"
```

Comment:
Think somewhere else you were converting string concatenations to f-strings, probably makes sense to do this here as well.